pkg/virt-controller/watch/topology/filter.go

```diff
@@ -10,6 +12,7 @@ import (
 const TSCFrequencyLabel = virtv1.CPUTimerLabel + "tsc-frequency"
 const TSCFrequencySchedulingLabel = "scheduling.node.kubevirt.io/tsc-frequency"
 const TSCScalableLabel = virtv1.CPUTimerLabel + "tsc-scalable"
+const TSCTolerancePPM = 250
```

Comment:
nit:
declaring `TSCTolerancePPM` as a float64 will save a type casting in line 98

Reply:
Oh, consts are typed, of course I knew that :sweat_smile: